app/mailers/community_mailer.rb

```diff
@@ -44,8 +44,8 @@ def open_prison_supporting_com_needed
 
   def assign_com_less_than_10_months
     set_template('6cae6890-6a5a-4ceb-82bd-43c8b43fc639')
-
-    set_personalisation(**params.slice(:prisoner_number, :prison_name, :crn_number, :prisoner_name))
+    params[:nomis_offender_id] = params.fetch(:prisoner_number)
+    set_personalisation(**params.slice(:prisoner_number, :nomis_offender_id, :prison_name, :crn_number, :prisoner_name))
```

Comment:
are `prisoner_number` and `nomis_offender_id` different things in this context?

Reply:
I'll refer you to the commit where I explained it :wink:
https://github.com/ministryofjustice/offender-management-allocation-manager/pull/2223/commits/99f1e98f49120f12f48d1c87441f071ebcd4f59a